### PR TITLE
Add Prime factors algorithm in java

### PR DESCRIPTION
**Pull request by Thuva4**

### CODE REVIEW

File: code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def binomialCoeff(n, k):
```
Comment (arnavb) [inline code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: Add OpenGenus comment.

File: code/mathematical-algorithms/factorial/factorial.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//factorial without using recursion
```
Comment (arnavb) [inline code/mathematical-algorithms/factorial/factorial.py]: Add OpenGenus comment and fix comment above to use a #, not a //

File: code/mathematical-algorithms/prime_factors/prime_factors.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import java.util.*;
```
Comment (arnavb) [inline code/mathematical-algorithms/prime_factors/prime_factors.java]: Add OpenGenus comment.

File: code/mathematical-algorithms/prime_factors/prime_factors.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        // if you want the duplicate remove the following 3 lines
+        Set<Integer> hs = new HashSet<>();
+        hs.addAll(al);
+        al.clear();
+        al.addAll(hs);
+        for (Integer integer : al) {
+            System.out.println(integer);
+        }
+    }
+}
```
Comment (yatharthshahjpr) [inline code/mathematical-algorithms/prime_factors/prime_factors.java]: Please add a new line after completing the code ! 👍

File: code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py (hunk 1 above)
Comment (yatharthshahjpr) [reply code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: Comment is  :-  /* Part of Cosmos by OpenGenus Foundation */ 👍

Comment (AdiChat) [reply code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: @yatharthshahjpr Kindly note your comment format is misleading.

Comment (AdiChat) [reply code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: @yatharthshahjpr Kindly note your comment format is misleading

File: code/mathematical-algorithms/prime_factors/prime_factors.java (hunk 4 above)
Comment (AdiChat) [reply code/mathematical-algorithms/prime_factors/prime_factors.java]: Any technical reason for this?

Comment (Thuva4) [reply code/mathematical-algorithms/prime_factors/prime_factors.java]: Sorry, I did not find this java implementation in repository.

Comment (AdiChat) [reply code/mathematical-algorithms/prime_factors/prime_factors.java]: @Thuva4 The concerned comment has been deleted by the other user. It was not intended on you. That user asked you to add newline at end of file.

Comment (AdiChat) [reply code/mathematical-algorithms/prime_factors/prime_factors.java]: @Thuva4 The concerned comment has been deleted by the other user (@yatharthshahjpr) . It was not intended on you. That user asked you to add newline at end of file.

**Comment by AdiChat** [comment]
The factorial code in `Python` exists. Kindly remove it 👍

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**